sqlitecluster/SQLiteNode.cpp

```diff
@@ -1258,6 +1258,14 @@ void SQLiteNode::_onMESSAGE(SQLitePeer* peer, const SData& message) {
             peer->version = message["Version"];
             peer->state = stateFromName(message["State"]);
 
+            // Is it on the same version as us?
+            if (!_haveSeenPeerOnSameVersion && peer->version.load() == _version) {
+                _haveSeenPeerOnSameVersion = true;
+                if (_haveBeenWAITING && _haveSeenPeerOnSameVersion) {
+                _priority = _originalPriority;
```

Comment:
Indentation?

Reply:
Similar to comment below, I think you can remove `&& _haveSeenPeerOnSameVersion` here because you set it to `true` right above